### ISSUE TITLE
GUID and ID of a player has a lot of aliasses and ip addresses

### ISSUE THREAD

**Issue by wassie** [MichaelHillcox/Echelon#7]
I'm not sure if this a Echelon issue or maybe a game or B3 issue. I have noticed that a lot of players have multiple aliasses and multiple ip addresses, sometimes more then 30. Is there a way to be sure that those aliasses are really the aliasses from that player? Is that a Echelon thing?

I don't know how the guid and id is made, so it could be my not knowing.....
By the way, this is on a Call of Duty UO server.

**Comment by wassie** [comment]
@MichaelHillcox do you know something about GUID and ID in relation to Echelon? is it normall that a user has so many ip's, from different countries?

**Comment by MichaelHillcox** [comment]
@wassie Hi,
So yeah the aliasses are because when people user /name it logs that name as a new alias. So for players you are finding with a lot of aliasses that is normal.

Hmm. The IP's are also because people can have dynamic IP's so there ip changes all the time. 

The B3 ID is assigned to the players GUID. so there is a new b3 user for each guid.

**Comment by wassie** [comment]
yes, I guessed right about the aliasses. but so many ip addresses with big different geographic places is weird. how relaible should that be?
And is the GUID something unique?

**Comment by MichaelHillcox** [comment]
The guid can be spoofed rather easily. Mainly because its just a 32bit code stored client side. The server will only check if the cd key is valid to allow a player in.

Not sure about ips though. I will be having these under a sub sections for extra nosiness but we have some people on my communities servers that have quite a lot but they are all that one players. Some people also use VPN's on there pc's so every new connect will be a completely random IP from somewhere in the world.
